### PR TITLE
Enhancement/visual-adjustments-home

### PR DESCRIPTION
**Pull request by javiermlovo**
Changes in the CSS to adjust the Home like the design
image

### CODE REVIEW

**Comment by tatarodriguez11** [comment]
por favor adjunta un pantallazo mobile, que fue donde se identificaron los issues visuales

File: styles/common/mainAllies.scss [hunk 1]
```diff
@@ -25,20 +25,23 @@
     grid-template-columns: repeat(2, 1fr);
     grid-gap: 22px;
     margin-bottom: 30px;
+    overflow: hidden;
```
Comment (Cruzlex7) [inline styles/common/mainAllies.scss:28]: In which cases something must be no visible.

Comment (javiermlovo) [reply styles/common/mainAllies.scss]: I changed my mind and limited the numbers of allies with slice(0, 18) instead of limit by CSS.

**Comment by javiermlovo** [comment]
image
The trouble with the responsive was the menu was sticky but in the background.

**Pull request closed without merging**